hs_communities/views/communities.py

```diff
@@ -37,10 +37,15 @@ def get_context_data(self, **kwargs):
             groups.append({'id': str(g.id), 'name': str(g.name), 'res_count': str(res_count)})
 
         groups = sorted(groups, key=lambda key: key['name'])
+
+        u = User.objects.get(pk=self.request.user.id)
+        is_admin = u.username == 'czo_national'
```

Comment:
how about a test and a TODO comment to clean this up in a refactor when we open up communities to other communities?

Reply:
I've added a comment here to capture the TODO https://github.com/hydroshare/hydroshare/issues/3622